app/views/compute_resources_vms/form/xenserver/_base.html.erb

```diff
@@ -72,125 +65,30 @@
-            $('#compute_attribute_vm_attrs_custom_template_name').attr("disabled", 'disabled');
-            $('.custom_template_name').prop("disabled", true);
-
-        }
-
-    });
-
-
-    // Host Profile Forms
-
-    // Check which template are in use
-    if ($("#host_compute_attributes_custom_template_name").val() != "") {
-        $('#host_compute_attributes_builtin_template_name').val($("#host_compute_attributes_builtin_template_name option:contains('No template')").val());
-        // IE and Chrome
-        $('#host_compute_attributes_builtin_template_name').attr("disabled", 'disabled');
-        $('.builtin_templates').prop("disabled", true);
-    }
-
-    if ($("#host_compute_attributes_builtin_template_name").val() != "") {
-        $('#host_compute_attributes_custom_template_name').val($("#host_compute_attributes_custom_template_name option:contains('No template')").val());
-        // IE and Chrome
-        $('#host_compute_attributes_custom_template_name').attr("disabled", 'disabled');
-        $('.custom_template_name').prop("disabled", true);
-    }
-
-    $(document).off('change.xenserver', '#host_compute_attributes_custom_template_name');
-    $(document).on('change.xenserver', '#host_compute_attributes_custom_template_name', function (data) {
-        if ($('option:selected', data.target).val() == "") {
-            $('.xenserver-raw').show(1000);
-            // IE and Chrome
-            $('.builtin_templates').prop("disabled", false);
-            $('#host_compute_attributes_builtin_template_name').removeAttr('disabled');
-        } else {
-            $('.xenserver-raw').hide(1000);
-            $('#host_compute_attributes_builtin_template_name').val($("#host_compute_attributes_builtin_template_name option:contains('No template')").val());
-            // IE and Chrome
-            $('#host_compute_attributes_builtin_template_name').attr("disabled", 'disabled');
-            $('.builtin_templates').prop("disabled", true);
-
-        }
-
-    });
-
-    $(document).off('change.xenserver', '#host_compute_attributes_builtin_template_name');
-    $(document).on('change.xenserver', '#host_compute_attributes_builtin_template_name', function (data) {
-        if ($('option:selected', data.target).val() == "") {
-            // IE and Chrome
-            $('.builtin_templates').prop("disabled", false);
-            $('#host_compute_attributes_custom_template_name').removeAttr('disabled');
-        } else {
-            $('#host_compute_attributes_custom_template_name').val($("#host_compute_attributes_builtin_template_name option:contains('No template')").val());
-            // IE and Chrome
-            $('#host_compute_attributes_custom_template_name').attr("disabled", 'disabled');
-            $('.custom_template_name').prop("disabled", true);
-
-        }
-    })
 </script>
-
-<%= compute_specific_js(compute_resource, 'foreman_xen/cache_refresh') %>
-<%= compute_specific_js(compute_resource, 'foreman_xen/populate_fields') %>
```

Comment:
@MrMEEE all these 100 LOC are really not needed anymore? If yes, the respective JS files in `app/assets/javascripts/foreman_xen/xenserver` can also get removed.

Reply:
Hi @mmoll 

I didn't write this patch, so hornestly, I don't know.. 

I just cleaned up the pull request as good as I could, and rebased..

Reply:
Somebody else should test this then, I for myself don't use this plugin, so I also can't tell...